test/blank.test.js

```diff
@@ -30,5 +30,11 @@ describe('#blank', () => {
         expect(wrapper.find('#parent')).to.be.blank()
       }).to.throw(`to be empty`)
     })
+
+    it('fails when actual is undefined', () => {
+      expect(() => {
+        expect(undefined).to.be.blank()
+      }).to.throw()
```

Comment:
Would use the same list of actuals [as above](https://github.com/producthunt/chai-enzyme/pull/11/files#r50515347):

```js
it('fails when the actual is not an enzyme wrapper'), () => {
  [undefined, { foo: 'bar' }, [], 'test', 12345].forEach((actual) => {
    expect(() => {
      expect(actual).to.be.blank()
    }).to.throw()
  })
})
```